src/ragas/metrics/_rubrics_based.py

```diff
@@ -0,0 +1,120 @@
+1. Write detailed feedback that assesses the quality of the response strictly based on the given score rubric, without evaluating in general.
+2. After writing the feedback, assign a score between 1 and 5, referring to the score rubric.""",
+    examples=[
+        {
+            "question": "What is the capital of France?",
+            "answer": "The capital of France is Paris.",
+            "ground_truth": "The capital of France is Paris.",
+            "rubrics": DEFAULT_RUBRICS,
+            "analysis": ScoreFeedbackAnswers.parse_obj(
+                [
+                    {
+                        "feedback": """The response is completely accurate and directly answers the question about the capital of France. It matches the reference answer perfectly and does not contain any errors or omissions. Given the rubric, this response deserves the highest score as it meets all the criteria for accuracy and clarity.""",
+                        "score": 5,
+                    }
+                ]
+            ).dicts(),
+        }
+    ],
+    input_keys=["question", "answer", "ground_truth", "rubrics"],
+    output_key="analysis",
+    language="english",
+)
+
+
+@dataclass
+class RubricsBasedScore(MetricWithLLM):
+    name: str = "absolute_rubrics_score"  # type: ignore
+    evaluation_mode: EvaluationMode = EvaluationMode.qga  # type: ignore
+    rubrics: t.Optional[t.Dict[str, str]] = None
+    scoring_prompt: Prompt = field(default_factory=lambda: SCORING_PROMPT)
+    max_retries: int = 1
+
+    def __post_init__(self):
+        self.rubrics = DEFAULT_RUBRICS if self.rubrics is None else self.rubrics
+
+    async def _ascore(self, row: t.Dict, callbacks: Callbacks) -> float:
+        assert self.llm is not None, "LLM is not set"
+
+        prompt_value = self._create_prompt(row)
+
+        response = await self.llm.generate(prompt_value, callbacks=callbacks)
+
+        parsed_response = await _score_feedback_output_parser.aparse(
+            response.generations[0][0].text, prompt_value, self.llm, self.max_retries
+        )
+
+        if parsed_response is None:
+            return np.nan
+
+        score = parsed_response.dicts()[0]["score"]
+        return score
+
+    def _create_prompt(self, row: t.Dict) -> PromptValue:
+        question, answer, ground_truth = (
+            row["question"],
+            row["answer"],
+            row["ground_truth"],
+        )
+        return self.scoring_prompt.format(
+            question=question,
+            answer=answer,
+            ground_truth=ground_truth,
+            rubrics=self.rubrics,
+        )
+
+    def adapt(self, language: str, cache_dir: t.Optional[str] = None) -> None:
+        assert self.llm is not None, "LLM must be set to adapt the metric"
+        self.scoring_prompt.adapt(language, self.llm, cache_dir)
+
+    def save(self, cache_dir: t.Optional[str] = None) -> None:
+        self.scoring_prompt.save(cache_dir)
+
+
+absolute_rubrics_score = RubricsBasedScore()
```

Comment:
just have one question here, should we use the name absolute_rubrics_score or prometheus score? I vote for prometheus so that it is easier to relate with the other project?